### PR TITLE
Add Transformation constructor for pipeline and fix show for it.

### LINKED ISSUE #69: Create Transformation with a single pipeline projection
Now to setup a pipeline, we need to handle the C API:

```julia
julia> pipe = Proj.proj_create("+proj=pipeline +step +proj=unitconvert +xy_in=deg +z_in=m +xy_out=rad +z_out=m +step +inv +proj=longlat +a=6378136.3 +rf=298.257 +e=0.08181922146 +step +proj=cart +a=6378136.3 +rf=298.257 +step +inv +proj=cart +ellps=WGS84 +step +proj=unitconvert +xy_in=rad +z_in=m +xy_out=deg +z_out=m +step +proj=axisswap +order=2,1")
Ptr{Nothing} @0x00000001254061d0

julia> Proj.proj_trans(pipe, Proj.PJ_FWD, (0,0,0))
4-element Proj.Coord:
  0.0
  0.0
 -0.7000000001862645
 Inf
```

### PR DESCRIPTION
**Pull request by evetion**
Fixes #69

### CODE REVIEW

File: src/coord.jl [hunk 1]
```diff
@@ -83,6 +86,17 @@ function Transformation(
     return Transformation(pj, direction)
 end
 
+function Transformation(
+    pipeline::AbstractString,
```
Comment (visr) [inline src/coord.jl:90]:
```suggestion
    pipeline::AbstractString;
```

File: src/coord.jl [hunk 2]
```diff
@@ -83,6 +86,17 @@ function Transformation(
...
+    always_xy::Bool = false,
+    direction::PJ_DIRECTION = PJ_FWD,
```
Comment (visr) [inline src/coord.jl:92]: Why is `area::Ptr{PJ_AREA} = C_NULL` not included like in the other constructors? Or does that not apply to pipelines?

**Review by visr — CHANGES_REQUESTED** [review]
This is great to have, thanks. I also like the Base.show improvements.

1. The new single arg AbstractString Transformation constructor is supposed to represent a pipeline. Can we check in the constructor after `proj_create` if this is really the case, and throw an error if not? Or is there no PROJ API for that? I'm worried about people creating a `Proj.Transformation("EPSG:28992")`, which (I guess) doesn't make much sense or will error.
2. Could you add a test for using the new constructor, including one that throws in the example above?

File: src/coord.jl (hunk 2 above)
Comment (evetion) [reply src/coord.jl:92]: Area is used to restrict the search for a pipeline between two CRS. Here we directly construct a pipeline, so no search is involved.

**Review by visr — APPROVED** [review]

**Pull request merged**